samples/wifi/radio_test/Kconfig.sysbuild

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF700X_RADIO_TEST_COMBO
+	bool "Enable radio test combo build for Wi-Fi and Bluetooth"
+	depends on SOC_NRF5340_CPUAPP
```

Comment:
```suggestion
```

Reply:
Could you provide reason for this suggestion. Chaitanya is not available today and I am looking for some context before making this change.

Thanks

Reply:
This isn't needed, in your app you would just add to sysbuild.conf (or sample.yaml): `SB_CONFIG_NETCORE_PERIPHERAL_RADIO_TEST=y`

Reply:
Sure, but what about no-sysbuilt support? As this is the first sysbuild release I intentionally left both versions of macros.WDYT?